File: myExpenses/src/androidTest/java/org/totschnig/myexpenses/test/util/AppDirHelperTest.java
package org.totschnig.myexpenses.test.util;


import org.junit.Test;
import org.junit.runner.RunWith;
import org.totschnig.myexpenses.util.AppDirHelper;

import androidx.test.InstrumentationRegistry;
import androidx.test.ext.junit.runners.AndroidJUnit4;

import static org.junit.Assert.assertTrue;

@RunWith(AndroidJUnit4.class)
public class AppDirHelperTest {
  @Test
  public void shouldValidateDefaultAppDir() {
    assertTrue(AppDirHelper.checkAppDir(InstrumentationRegistry.getTargetContext()).isSuccess());
  }
}

File: myExpenses/src/androidTest/java/org/totschnig/myexpenses/test/util/AppDirHelperTest.kt
package org.totschnig.myexpenses.test.util

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.platform.app.InstrumentationRegistry
import org.junit.Assert
import org.junit.Test
import org.junit.runner.RunWith
import org.totschnig.myexpenses.util.AppDirHelper.checkAppDir

@RunWith(AndroidJUnit4::class)
class AppDirHelperTest {
    @Test
    fun shouldValidateDefaultAppDir() {
        Assert.assertTrue(checkAppDir(InstrumentationRegistry.getInstrumentation().targetContext).isSuccess)
    }
}